docs/Special/RoNeX-on-Windows-VirtualBox-and-MATLAB.rst

```diff
@@ -121,14 +121,15 @@ workspace, named rostest.m
 
 .. code-block:: matlab
 
-    node = rosmatlab.node('my_data_listener_node','http://192.168.0.14:11311');
-    subscriber = rosmatlab.subscriber('/ronex/general_io/12/state','sr_ronex_msgs/GeneralIOState',10,node);
-    subscriber.setOnNewMessageListeners({@my_function});
+    rosinit('http://192.168.0.14:11311')
+    sub = rossubscriber('/ronex/general_io/1409578750/state', @my_function, 'Buffersize', 10))
```

Comment:
Maybe an alias. But that means that the user has defined an alias.
Also I thought that 12 wasn't a good example, as the serial numbers look more like `1409578750`.